examples/persistence_service/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# (c) 2022 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-persistence-service)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../resources/cmake"
+)
+
+include(ConnextDdsAddExamplesSubdirectories)
+
+connextdds_add_examples_subdirectories(
```

Comment:
```suggestion
connextdds_add_example_subdirectories(
```
Is that a function you're reusing? We should rename that function everywhere it's used.

Reply:
This function was created by @ManuelJNunez so targets for each language subdirectories are added. It also enables you to add many examples with just one call to this function, therefo0re the plural of the name.

Reply:
We'll address this a separate issue.